src/Data/FastMutableIntMap.hs

```diff
@@ -91,3 +92,6 @@ applyPatch (FastMutableIntMap r) p@(PatchIntMap m) = do
   v <- readIORef r
   writeIORef r $! applyAlways p v
   return $ IntMap.intersection v m
+
+toList :: FastMutableIntMap a -> IO [(Int, a)]
+toList (FastMutableIntMap r) = IntMap.toList <$> readIORef r
```

Comment:
Would `readToList`, `readList`, or `readAll` be better?

Reply:
probably not actually; very little of the interface makes such distinctions.